pyronear/datasets/wildfire/frame_extractor.py

```diff
@@ -71,6 +71,7 @@ def __init__(self,
                              f"Please choose from : {', '.join(self.strategies_allowed)}")
 
         self.states = pd.read_csv(path_to_states)
+        this.states = this.states.astype({"stateStart": int, "stateEnd": int})
```

Comment:
As it is documented in the [FrameExtractor docstring](https://github.com/x0s/PyroNear/blob/add-wildfire-frame-extractor/pyronear/datasets/wildfire/frame_extractor.py#L40) and in the tests [with states.csv sample](https://github.com/x0s/PyroNear/blob/add-wildfire-frame-extractor/test/fixtures/wildfire_states.csv) but also [here](https://github.com/x0s/PyroNear/blob/add-wildfire-frame-extractor/test/test_datasets_wildfire_frame_extractor.py#L61), [and here](https://github.com/x0s/PyroNear/blob/add-wildfire-frame-extractor/test/test_datasets_wildfire_frame_extractor.py#L46), [there](https://github.com/x0s/PyroNear/blob/add-wildfire-frame-extractor/test/test_datasets_wildfire_frame_extractor.py#L78) and [there](https://github.com/x0s/PyroNear/blob/add-wildfire-frame-extractor/test/test_datasets_wildfire_frame_extractor.py#L89), it is expected to have ints in the CSV since the states are integer by definition. It is then reasonable to expect that another function that generate such CSV would export the statesStart and stateEnd features as integers not float.

This being said, it makes no arm to enforce this type here, but I would recommend to also cast to integers when exporting this features to csv if that's not the case.

About the implementation, Instead to reading twice these columns to import then to cast type, you directly can do it when reading the CSV this way:
```python
self.states = pd.read_csv(path_to_states, dtype={"stateStart": int, "stateEnd": int})
```

Reply:
Since, I'm the owner of this PR, I cannot request changes. 
To prevent the PR to be merged inadvertently without taking into account my request changes, I converted the PR to draft waiting.

That's one more reason to avoid adding a new feature in a PR in review state. It's not big deal, though, Paradoxically, I will mark it again as ready for review once the "owner"'s review would be done ^^

Reply:
@MateoLostanlen would you mind taking a look at it please? I'll request changes on behalf of @x0s

Reply:
Hi ! This is a typing error but I have already correct it in the following commit :
https://github.com/frgfm/PyroNear/pull/48/commits/2f8fd23bce207f977162f33438cd10cff33950f2

Reply:
@MateoLostanlen : My review was not about the typing error, maybe read it again ;) 
https://github.com/frgfm/PyroNear/pull/48#discussion_r415638269

Reply:
Okay, sorry, I read that a little fast.

Reply:
Perhaps @x0s you might want to add commit to revert the changes of @MateoLostanlen on this section so we can merge the PR?
Or is there anything else blocking?

Reply:
@frgfm: As I pointed on slack, since my motherboard became defective, I have limited access to the repository.
Nevertheless, last week, I recommended  to @MateoLostanlen to revert its commits in favor of a new PR if needed. For instance:
```
git revert --no-commit b2fefbc^..34a506c
git commit -m "Revert reviewer commits in favor of another PR"
```
What else is blocking ?

Reply:
I'm good with that, it was just to avoid a multi-author PR situation. But yes, @MateoLostanlen could you revert this change please?

Reply:
@frgfm : I don't get what you mean by "multi-author PR situation", can you elaborate ?